sentences/cs/light_HassLightSet.yaml

```diff
@@ -2,4 +2,23 @@ language: cs
 intents:
   HassLightSet:
     data:
-    - sentences: []
+      #jas | brightness
```

Comment:
Maybe I'm having my OCD moment here, but would be happier if there was a space between # and jas in the same way as you did for # barva | color.